grammar/src/leo.pest

```diff
@@ -454,18 +454,25 @@ input_tuple = _{ "(" ~ (input ~ ("," ~ input)* ~ ","?)? ~ ")"}
 /// Imports
 
 // Declared in imports/import.rs
-import = { "import " ~ package ~ LINE_END}
+import = { "import " ~ package_type ~ LINE_END}
```

Comment:
You might consider using `package_or_packages` instead of `package_type`, to avoid any potential confusion with Leo types.

Reply:
+1 on this, it seems that could help improve readability